.changeset/github-channel-support.md

```diff
@@ -0,0 +1,5 @@
+---
+"electron-updater": patch
```

Comment:
This should be `minor` for a `feat` introduction :)

Reply:
Thanks, Changed.